Tensile/TensileCreateLib/ParseArguments.py

```diff
@@ -0,0 +1,312 @@
+    warnings.showwarning = showwarning
+
+    parser = ArgumentParser(
+        description="TensileCreateLibrary generates libraries and code object files "
+        "for a set of supplied logic files.",
+    )
+
+    # Positional arguments
+    parser.add_argument("LogicPath", help="Path to Library Logic (YAML) files.")
+    parser.add_argument("OutputPath", help="Build path for library files.")
+    parser.add_argument(
+        "RuntimeLanguage",
+        help="Runtime langauge for generated library.",
+        choices=["OCL", "HIP", "HSA"],
+    )
+
+    # Optional arguments
+    compilerChoices = ["amdclang++", "hipcc"] if os.name != "nt" else ["clang++", "hipcc"]
+    parser.add_argument(
+        "--cxx-compiler",
+        dest="CxxCompiler",
+        choices=compilerChoices,
+        default=compilerChoices[0],
+        type=str,
+        help="C++ compiler used when generating binaries."
+        " On linux, amdclang++ (default) or hipcc. On Windows clang++ (default) or hipcc.",
+    )
+    parser.add_argument(
+        "--architecture",
+        dest="Architecture",
+        default="all",
+        type=str,
+        help="Architectures to generate a library for. When specifying multiple options, "
+        "use quoted, semicolon delimited architectures, e.g., --architecture='gfx908;gfx1012'. "
+        "Supported archiectures include: " + ", ".join(architectureMap.keys()),
+    )
+    parser.add_argument(
+        "--code-object-version",
+        dest="CodeObjectVersion",
+        choices=["default", "V4", "V5"],
+        type=str,
+        help="HSA code-object version.",
+    )
+    parser.add_argument(
+        "--cmake-cxx-compiler",
+        dest="CmakeCxxCompiler",
+        action=DeprecatedOption,
+        help="(Deprecated) Set the environment variable CMAKE_CXX_COMPILER.",
+    )
+
+    # Boolean flags
+    parser.add_argument(
+        "--merge-files",
+        dest="MergeFiles",
+        action="store_true",
+        help="Store all solutions in single file.",
+    )
+    parser.add_argument(
+        "--no-merge-files",
+        dest="MergeFiles",
+        action="store_false",
+        help="Store every solution and kernel in separate file.",
+    )
+    parser.add_argument(
+        "--num-merged-files",
+        dest="NumMergedFiles",
+        default=1,
+        type=int,
+        help="Number of files the kernels should be written into.",
+    )
+    parser.add_argument(
+        "--short-file-names",
+        dest="ShortNames",
+        action="store_true",
+        help="Converts solution and kernel names to serial IDs.",
+    )
+    parser.add_argument(
+        "--no-short-file-names",
+        dest="ShortNames",
+        action="store_false",
+        help="Disables short files names.",
+    )
+    parser.add_argument(
+        "--library-print-debug",
+        dest="LibraryPrintDebug",
+        type=bool,
+        default=True,
+        action=DeprecatedOption,
+        help="(Deprecated)",
+    )
```

Comment:
We can remove this altogether.

Reply:
Also removing `--no-library-print-debug`